hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -172,4 +173,18 @@ void Exception(int errorCode, String className, @Nullable String message, StackT
     @Response(ResponseMessageConst.ENTRIES_WITH_CURSOR)
     void EntriesWithCursor(int tableIndex, List<Map.Entry<Data, Data>> entries);
 
+    @Since("1.4")
+    @Response(ResponseMessageConst.SCHEDULED_TASK_HANDLER)
+    void ScheduledTaskHandler(ScheduledTaskHandler handler);
```

Comment:
Why not do this: Get rid of ScheduledTaskHandlerCodec and write here:
void ScheduledTaskHandler(String urn);